python/lsst/ip/diffim/subtractImages.py

```diff
@@ -581,6 +576,42 @@ def finalize(self, template, science, difference, kernel,
             correctedExposure = difference
         return correctedExposure
 
+    def updateMasks(self, difference, science, template):
```

Comment:
Could you keep the same order of images as in `finalize`: template, science, difference

Reply:
Done!